jpegexport.py

```diff
@@ -122,76 +125,75 @@ def processSelected(self):
         
 
         if self.options.fast == True:  # uses simpletransform
-            bbox = simpletransform.computeBBox(nodelist)
-            startx = math.ceil(bbox[0])
-            endx = math.ceil(bbox[1])
-            h = -bbox[2] + bbox[3]
-            starty = toty - math.ceil(bbox[2]) -h
-            endy = toty - math.ceil(bbox[2])
+            bbox = sum([node.bounding_box() for node in nodelist], None)
```

Comment:
Isn't there a function for this in inkex already?

Reply:
hmm i updated this using the provided tip from deprecation warning. did not find some shorter line for this